server/apps/io/reuters.py

```diff
@@ -102,9 +102,12 @@ def get_ids(self, channel, last_updated, updated):
         ids = []
         payload = {'channel': channel, 'fieldsRef': 'id'}
         payload['dateRange'] = "%s-%s" % (self.format_date(last_updated), self.format_date(updated))
+        logger.info('Reuters requesting Date Range |{}| for channel {}'.format(payload['dateRange'], channel))
         tree = self.get_tree('items', payload)
         for result in tree.findall('result'):
-            ids.append(result.find('guid').text)
+            id = result.find('guid').text
+            if id not in ids:
+                ids.append(id)
```

Comment:
The list can be up to 300 items (not unique) or maybe 220 unique items on initial start-up when it attempts to request 24Hrs of content.  Usually when polling it would only be a couple. I guess 300 is a significant number so I'll change it to use a set, it will simplify the code!